components/Shared/Input/Number.jsx

```diff
@@ -57,6 +57,14 @@ export const NumberInput = forwardRef(
       setError,
       valid,
       fontSize,
+      border,
+      color,
+      borderTop,
+      borderBottom,
+      denomBorderTopLeftRadius,
```

Comment:
Why is it better to pass these as props rather than just hardcode them here? Do you foresee someone else using the Shared Number input wanting to change all the border radiuses of the denom tag? 

Seems to me that the border radiuses of the denom tags would be something developers wouldn't care about

Reply:
Yeah this was something I wanted your opinion on and telegraphed this the other day. 

It's really a question of my ignorance and not knowing what "best practices" are in this regard. E.g. Should we:

1.  Allow for components to be styled where they are used, to improve visibility on what styles have been declared and simplify component style customizations
1. Keep all styling down inside the component file.

Reply:
It helps to reason about when you think about the entire `/components/Shared` directory as its own UI library. Then you can ask yourself the golden question:

If I'm a developer importing this library from npm, what styles do I care to adjust myself? Which do I want taken care of for me? Sometimes its hard to think about that (i'm sure esp as a designer), but basically, if we want to allow outside developers to adjust the styles, then we should create props for those stylistic elements. If we want to take care of the styling for the developers, we should put all that code inside the shared component.

Since I dont think developers care about the border radius of the denom tag in the Number input, I would vote to keep that styling in the DenomTag itself. As a fallback, a developer can always use the BaseInput and style that him/herself